RabbitMQ/config.blade.php

```diff
@@ -1,19 +1,19 @@
 <h2>{{ __('app.apps.config') }} ({{ __('app.optional') }}) @include('items.enable')</h2>
 <div class="items">
-    <input type="hidden" data-config="dataonly" class="config-item" name="config[dataonly]" value="1" />
-    <div class="input">
-        <label>{{ strtoupper(__('app.url')) }}</label>
-        {!! Form::text('config[override_url]', isset($item) ? $item->getconfig()->override_url : null, ['placeholder' => __('app.apps.override'), 'id' => 'override_url', 'class' => 'form-control']) !!}
-    </div>
+	<input type="hidden" data-config="dataonly" class="config-item" name="config[dataonly]" value="1" />
+	<div class="input">
+		<label>{{ strtoupper(__('app.url')) }}</label>
+		{!! Form::text('config[override_url]', isset($item) ? $item->getconfig()->override_url : null, ['placeholder' => __('app.apps.override'), 'id' => 'override_url', 'class' => 'form-control']) !!}
+	</div>
 	<div class="input">
 		<label>Username</label>
-		{!! Form::input('text', 'config[username]', isset($item) ? $item->getconfig()->username : null, ['placeholder' => __('Username'), 'data-config' => 'username', 'class' => 'form-control config-item']) !!}
+		{!! Form::text('config[username]', isset($item) ? $item->getconfig()->username : null, ['placeholder' => __('app.apps.username'), 'data-config' => 'username', 'class' => 'form-control config-item']) !!}
```

Comment:
Neat thanks, took most of this from the HA app :+1:

Reply:
Yeah A bunch of the apps did things slightly different so I have been standardising them :)

Reply:
That is good to know as I'm also planning to PR the Unifi app into an enhanced app :)